cob_bringup/components/canopen_base.launch

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<launch>
+    <arg name="robot"/>
+    <arg name="component_name" default="base"/>
+    <arg name="can_device" default="can0"/>
+    <include file="$(find cob_bringup)/drivers/canopen_402.launch">
+        <arg name="robot" value="$(arg robot)"/>
+        <arg name="component_name" value="$(arg component_name)"/>
+        <arg name="can_device" value="$(arg can_device)"/>
+    </include>
+    <include file="$(find cob_bringup)/controllers/base_controller_plugin.launch">
+        <arg name="robot" value="$(arg robot)"/>
+    </include>
```

Comment:
Yes, a ```sim``` argument might help....e.g. sick_s300.launch is a candidate, too, as it includes the driver for the laser scanner (hw-specific) but also the laser_scan_filter.launch (hw-independent) which is used by simulation, too....
(Another one would e.g. be light.launch)
Didn't put this in the Concept-PR as I just wanted to hear opinions about the general concept....